usr/share/rear/lib/framework-functions.sh

```diff
@@ -130,7 +130,7 @@ function cleanup_build_area_and_end_program () {
         # line below put in comment due to issue #465
         #rm -Rf $BUILD_DIR/outputfs
         # in worst case it could not umount; so before remove the BUILD_DIR check if above outputfs is gone
-        mount | grep -q "$BUILD_DIR/outputfs"
+        mountpoint -q "$BUILD_DIR/outputfs"
         if [[ $? -eq 0 ]]; then
```

Comment:
While you are at it I would like to suggest to clean it up a bit more
and simplify the code into a single line
```
  if mountpoint -q "$BUILD_DIR/outputfs" ; then
```

By the way:
Via
```
user@host:~/rear/rear-master$ find . | xargs grep 'mount .* grep '
```
I found another place with same kind of code
https://github.com/rear/rear/blob/master/usr/share/rear/verify/DUPLICITY/default/200_check_tmpfs.sh
that could be cleaned up as follows
```
...
# mount tmpfs on /tmp if not present
mountpoint -q /tmp && return 0
LogPrint "File system /tmp not present - mounting it via tmpfs"
mount -t tmpfs tmpfs /tmp || Error "Failed to mount tmpfs on /tmp"
```